components/locid/src/extensions/unicode/mod.rs

```diff
@@ -75,6 +75,7 @@ use litemap::LiteMap;
 /// ```
 #[derive(Clone, PartialEq, Eq, Debug, Default, Hash, PartialOrd, Ord)]
 #[allow(missing_docs)] // TODO(#1028) - Add missing docs.
+#[non_exhaustive]
```

Comment:
Suggestion: this is a spec-backed data structure. I feel that it should most likely be stable. @zbraniecki ?